mantidproject/mantid
# 6.5 release notes index page

**Description of work.**

Created a first draft of the index page, highlighting features which stood out to me as more major.

I have created a collage of three of the more visual changes. In the last release there was an image after the opening paragraph and one to the right of the table of contents; I have added the collage in both places for now. I'd like the smaller image by the table of contents to be something different but unsure as to what.

**To test:**

- Remove build/docs/doctrees
- Build the documents again and check there are no warnings
- Check that the links work
- Check the information is still correct
- Grammar/Spelling check
- Check images display properly

*There is no associated issue.*



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
